code/game/objects/items/devices/scanners.dm

```diff
@@ -81,8 +82,11 @@ HALOGEN COUNTER	- Radcount on mobs
 	dat += 		"\tKey: <font color='cyan'>Suffocation</font>/<font color='green'>Toxin</font>/<font color='#FFA500'>Burns</font>/<font color='red'>Brute</font><br>"
 	dat += 		"\tDamage Specifics: <font color='cyan'>[OX]</font> - <font color='green'>[TX]</font> - <font color='#FFA500'>[BU]</font> - <font color='red'>[BR]</font><br>"
 	dat +=		"Body Temperature: [M.bodytemperature-T0C]&deg;C ([M.bodytemperature*1.8-459.67]&deg;F)</span><br>"
-	if(M.tod && (M.stat == DEAD || (M.status_flags & FAKEDEATH)))
-		dat += 	"<span class='notice'>Time of Death: [M.tod]</span><br>"
+	if(M.timeofdeath && (M.stat == DEAD || (M.status_flags & FAKEDEATH)))
+		dat += 	"<span class='notice'>Time of Death: [worldtime2stationtime(M.timeofdeath)]</span><br>"
+		var/tdelta = round(world.time - M.timeofdeath)
+		if(tdelta < (DEFIB_TIME_LIMIT * 10))
+			dat += "<span class='danger'><b>Subject died [DisplayTimeText(tdelta)] ago- defibrillation may be possible!</b></span><br>"

```

Comment:
Idea as a whole good. But just few things about the message sent:

I think that there should be space between ago and -

Resuscitation is better used rather than defibrillation, since its more accurate term and term used even by defibrillators themselves

Also, for health analyzers printout, I think span class notice would be more fitting for that message than danger? But thats more of an opinion.

Reply:
:+1: implemented

Reply:
👍